webapp/packages/plugin-sql-editor-navigation-tab/src/ACTION_TAB_CLOSE_GROUP.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { createAction } from '@cloudbeaver/core-view';
+
+export const ACTION_TAB_CLOSE_GROUP = createAction('tab-close-group', {
+  label: 'ui_close_group',
+  tooltip: 'ui_close_group',
+});
```

Comment:
please move it to plugin-sql-editor and rename to ACTION_TAB_CLOSE_SQL_RESULT_GROUP

Reply:
also dont forget to move localization